package.json

```diff
@@ -6,10 +6,50 @@
     "node": "10.16.0"
   },
   "dependencies": {
-    "@babel/core": "7.5.5",
     "@material-ui/core": "^4.2.1",
     "@material-ui/icons": "^4.2.1",
-    "@svgr/webpack": "4.3.2",
+    "dotenv": "6.2.0",
```

Comment:
No `dotenv-flow`? 😢

Reply:
Maybe in the future, if it is really needed